Maven release fails under jdk8

```bash
$ mvn release:perform
...
[INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project ngs-align: MavenReportException: Error while creating archive:
[INFO] [ERROR] Exit code: 1 - /home/xxx/working/ngs/target/checkout/align/src/main/java/org/nmdp/ngs/align/BedRecord.java:47: error: self-closing element not allowed
[INFO] [ERROR] *   <li>BED3: A BED file where each feature is described by chrom, start, and end.<br/>
[INFO] [ERROR] ^
[INFO] [ERROR] /home/xxx/working/ngs/target/checkout/align/src/main/java/org/nmdp/ngs/align/BedRecord.java:49: error: self-closing element not allowed
[INFO] [ERROR] *   <li>BED4: A BED file where each feature is described by chrom, start, end, and name.<br/>
[INFO] [ERROR] ^
[INFO] [ERROR] /home/xxx/working/ngs/target/checkout/align/src/main/java/org/nmdp/ngs/align/BedRecord.java:51: error: self-closing element not allowed
[INFO] [ERROR] *   <li>BED5: A BED file where each feature is described by chrom, start, end, name, and score.<br/>
[INFO] [ERROR] ^
[INFO] [ERROR] /home/xxx/working/ngs/target/checkout/align/src/main/java/org/nmdp/ngs/align/BedRecord.java:53: error: self-closing element not allowed
[INFO] [ERROR] *   <li>BED6: A BED file where each feature is described by chrom, start, end, name, score, and strand.<br/>
[INFO] [ERROR] ^
[INFO] [ERROR] /home/xxx/working/ngs/target/checkout/align/src/main/java/org/nmdp/ngs/align/BedRecord.java:55: error: self-closing element not allowed
[INFO] [ERROR] *   <li>BED12: A BED file where each feature is described by all twelve columns listed above.<br/>
[INFO] [ERROR] ^
[INFO] [ERROR] /home/xxx/working/ngs/target/checkout/align/src/main/java/org/nmdp/ngs/align/BedRecord.java:57: error: unexpected end tag: </p>
[INFO] [ERROR] * </ul></p>
[INFO] [ERROR] ^
[INFO] [ERROR] 
[INFO] [ERROR] Command line was: /usr/lib/jvm/java-8-oracle/jre/../bin/javadoc @options @packages
```

--- a/pom.xml
+++ b/pom.xml
@@ -296,12 +296,13 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
-          <version>2.9.1</version>
+          <version>2.10.3</version>
           <configuration>
             <detectLinks>true</detectLinks>
             <quiet>true</quiet>
             <source>${maven.compile.source}</source>
             <linksource>true</linksource>
+            <additionalparam>-Xdoclint:none</additionalparam>
           </configuration>
         </plugin>
         <plugin>

